internals/utils/src/index.ts

```diff
@@ -0,0 +1,393 @@
+export * from './deploy/get-prebuilt-json';
+export {default as getPrebuiltJson} from './deploy/get-prebuilt-json';
+export * from './deploy/get-preferred-preview-url';
+export * from './deploy/parse-target';
+export {default as parseTarget} from './deploy/parse-target';
+export * from './deploy/process-deployment';
+export {default as processDeployment} from './deploy/process-deployment';
+export * from './deploy/validate-archive-format';
+export * from './env/add-env-record';
+export {default as addEnvRecord} from './env/add-env-record';
+export * from './env/diff-env-files';
+export * from './env/env-target';
+export * from './env/format-env-target';
+export {default as formatEnvTarget} from './env/format-env-target';
+export * from './env/get-env-records';
+export {default as getEnvRecords} from './env/get-env-records';
+export * from './env/known-error';
+export * from './env/remove-env-record';
+export {default as removeEnvRecord} from './env/remove-env-record';
+export * from './domains/add-domain';
+export {default as addDomain} from './domains/add-domain';
+export * from './domains/check-transfer';
+export {default as checkTransfer} from './domains/check-transfer';
+export * from './domains/get-auth-code';
+export {default as getAuthCode} from './domains/get-auth-code';
+export * from './domains/get-domain-by-name';
+export {default as getDomainByName} from './domains/get-domain-by-name';
+export * from './domains/get-domain-config';
+export * from './domains/get-domain-price';
+export {default as getDomainPrice} from './domains/get-domain-price';
+export * from './domains/get-domain-registrar';
+export * from './domains/get-domain-status';
+export {default as getDomainStatus} from './domains/get-domain-status';
+export * from './domains/get-domain';
+export * from './domains/get-domains';
+export {default as getDomains} from './domains/get-domains';
+export * from './domains/is-domain-external';
+export {default as isDomainExternal} from './domains/is-domain-external';
+export * from './domains/is-public-suffix';
+export * from './domains/maybe-get-domain-by-name';
+export {default as maybeGetDomainByName} from './domains/maybe-get-domain-by-name';
+export * from './domains/move-out-domain';
+export {default as moveOutDomain} from './domains/move-out-domain';
+export * from './domains/purchase-domain-if-available';
+export {default as purchaseDomainIfAvailable} from './domains/purchase-domain-if-available';
+export * from './domains/purchase-domain';
+export {default as purchaseDomain} from './domains/purchase-domain';
+export * from './domains/remove-domain-by-name';
+export {default as removeDomainByName} from './domains/remove-domain-by-name';
+export * from './domains/set-custom-suffix';
+export {default as setCustomSuffix} from './domains/set-custom-suffix';
+export * from './domains/setup-domain';
+export {default as setupDomain} from './domains/setup-domain';
+export * from './domains/transfer-in-domain';
```

Comment:
I think this is not solving the problem we set out to solve: 

I thought the goal was to move everything into separate internal packages so that turborepo only need to run tests for the feature that was modified. This PR looks like changing "vc domains" will still run test for `vc env` and vice-versa because they're in the same package.

Reply:
`package/cli` is a very complicated mess of inter-dependent code. I originally did attempt to only move the pieces required for `dev`, but then some file 3+ `import` calls deep somehow depended on some random piece of logic in another commands files. 

Simply moving every util file out of the cli package seems to be the best first step here to untangeling this mess with the least amount of code changes.